src/jumpstart/slice.ts

```diff
@@ -0,0 +1,55 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export interface State {
+  showLoading: boolean
+  showError: boolean
```

Comment:
Just checking, do we need both loading and error state at the same time?
If not, I'd advise to model it differently to avoid impossible states.

For instance: https://github.com/valora-inc/wallet/blob/2b8972311cd1da141b1bb09f5992bb99c783648e/src/positions/slice.ts#L6

Reply:
good point, thanks! refactored it to have just one consistent property:
```
export interface State {
  claimStatus: 'idle' | 'loading' | 'error'
}
```